aspnetcore/mvc/razor-pages/razor-pages-convention-features.md

```diff
@@ -40,7 +40,7 @@ The sample app adds a `{globalTemplate?}` route template to all of the pages in
 > [!NOTE]
 > The `Order` property for the `AttributeRouteModel` is set to `0` (zero). This ensures that this template is given priority for the first route data value position when a single route value is provided. For example, the sample adds an `{aboutTemplate?}` route template later in the topic. The `{aboutTemplate?}` template is given an `Order` of `1`. When the About page is requested at `/About/RouteDataValue`, "RouteDataValue" is loaded into `RouteData.Values["globalTemplate"]` (`Order = 0`) and not `RouteData.Values["aboutTemplate"]` (`Order = 1`) due to setting the `Order` property.
 
-*Startup.cs*:
+Razor Pages options, such as adding [Conventions](/dotnet/api/microsoft.aspnetcore.mvc.razorpages.razorpagesoptions.conventions), are added when MVC is added to the service collection in `Startup.ConfigureServices`. For an example, see the [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/mvc/razor-pages/razor-pages-convention-features/sample/).
```

Comment:
For 2.1.0-preview2, we also let you annotate types with attributes that implement a convention `class MyConventionAttribute : Attribute, IPagePropertyModelConvention`, but for `dev` this is correct

Reply:
@guardrex If you haven't already, please create an issue to track this change ☝️ in 2.1.0-preview2.